src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpProj.cs

```diff
@@ -3,75 +3,25 @@
 
 using System;
 using System.IO;
-using System.Linq;
 using System.Reflection;
-using System.Threading.Tasks;
 using AutoRest.CSharp.AutoRest.Communication;
-using AutoRest.CSharp.Input;
-using AutoRest.CSharp.Output.Models.Types;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using static System.Net.Mime.MediaTypeNames;
+using AutoRest.CSharp.Common.Input;
+using AutoRest.CSharp.Common.Output.Builders;
 
 namespace AutoRest.CSharp.AutoRest.Plugins
 {
-    // ReSharper disable once StringLiteralTypo
-    [PluginName("csharpproj")]
-    // ReSharper disable once IdentifierTypo
-    internal class CSharpProj : IPlugin
+    internal class CSharpProj
```

Comment:
I believe there should be follow ups of this PR to eventually remove this class and combine its functionality into `NewProjectScaffolding` because they have overlapped responsibilities.

That is kind of a big change in our source code, and the main purpose of this PR is to remove the `csharpproj` plugin to fix the concurrent issue we found. Other parts work fine - therefore we could have other PRs to refactor that part.